packages/async/src/services/asyncjobservice.ts

```diff
@@ -221,11 +244,102 @@ export default class AsyncJobService<T extends AsyncJobServiceParameters = Async
     return this.queue.sendMessage({ uuid: action.getUuid(), __secretKey: action.__secretKey, type: action.type });
   }
 
+  /**
+   * Return headers to request status hook
+   *
+   * @param jobInfo
+   * @returns
+   */
+  getHeaders(jobInfo: JobInfo) {
+    let res = {
+      "X-Job-Id": jobInfo.JOB_ID,
+      "X-Job-Time": Date.now().toString()
+    };
+    res["X-Job-Hash"] = crypto
```

Comment:
How can we use this value to request info later on the job  ?

Reply:
It depends on the runner implementation: https://github.com/loopingz/webda.io/blob/59a3837aed854d9ae3b72ffe34af1be3741443d9/packages/kubernetes/src/services/kuberunner.ts#L92
https://github.com/loopingz/webda.io/blob/59a3837aed854d9ae3b72ffe34af1be3741443d9/packages/async/src/services/localrunner.ts#L59
Both my runners push it with environment variables, so from the job you can call the hook to update your status.
As action is a CoreModel object for the user to read it, you should use the `canAct` provided by the store and expose the store

Reply:
https://docs.webda.io/pages/Concepts/AsyncActions.html